data-request/dreq_CORDEX-CORE.csv

```diff
@@ -0,0 +1,548 @@
+ps,1hr,Pa,Surface Air Pressure,surface_air_pressure,area: mean time: point,CORE,
+psl,mon,Pa,Sea Level Pressure,air_pressure_at_mean_sea_level,area: time: mean,CORE,
+psl,day,Pa,Sea Level Pressure,air_pressure_at_mean_sea_level,area: time: mean,CORE,
+psl,1hr,Pa,Sea Level Pressure,air_pressure_at_mean_sea_level,area: mean time: point,CORE,
+sfcWind,mon,m s-1,Near-Surface Wind Speed,wind_speed,area: time: mean,CORE,
+sfcWind,day,m s-1,Near-Surface Wind Speed,wind_speed,area: time: mean,CORE,
+sfcWind,1hr,m s-1,Near-Surface Wind Speed,wind_speed,area: mean time: point,CORE,
+uas,mon,m s-1,Eastward Near-Surface Wind,eastward_wind,area: time: mean,CORE,
+uas,day,m s-1,Eastward Near-Surface Wind,eastward_wind,area: time: mean,CORE,
+uas,1hr,m s-1,Eastward Near-Surface Wind,eastward_wind,area: mean time: point,CORE,
+vas,mon,m s-1,Northward Near-Surface Wind,northward_wind,area: time: mean,CORE,
+vas,day,m s-1,Northward Near-Surface Wind,northward_wind,area: time: mean,CORE,
+vas,1hr,m s-1,Northward Near-Surface Wind,northward_wind,area: mean time: point,CORE,
+clt,mon,%,Total Cloud Cover Percentage,cloud_area_fraction,area: time: mean,CORE,
+clt,day,%,Total Cloud Cover Percentage,cloud_area_fraction,area: time: mean,CORE,
+clt,1hr,%,Total Cloud Cover Percentage,cloud_area_fraction,area: time: mean,CORE,
+rsds,mon,W m-2,Surface Downwelling Shortwave Radiation,surface_downwelling_shortwave_flux_in_air,area: time: mean,CORE,
+rsds,day,W m-2,Surface Downwelling Shortwave Radiation,surface_downwelling_shortwave_flux_in_air,area: time: mean,CORE,
+rsds,1hr,W m-2,Surface Downwelling Shortwave Radiation,surface_downwelling_shortwave_flux_in_air,area: time: mean,CORE,
+rlds,mon,W m-2,Surface Downwelling Longwave Radiation,surface_downwelling_longwave_flux_in_air,area: time: mean,CORE,
+rlds,day,W m-2,Surface Downwelling Longwave Radiation,surface_downwelling_longwave_flux_in_air,area: time: mean,CORE,
+rlds,1hr,W m-2,Surface Downwelling Longwave Radiation,surface_downwelling_longwave_flux_in_air,area: time: mean,CORE,
+orog,fx,m,Surface Altitude,surface_altitude,area: mean,CORE,
+sftlf,fx,%,Percentage of the Grid Cell Occupied by Land,land_area_fraction,area: mean,CORE,
+ts,mon,K,Surface Temperature,surface_temperature,area: time: mean,TIER1,
+ts,day,K,Surface Temperature,surface_temperature,area: time: mean,TIER1,
+ts,1hr,K,Surface Temperature,surface_temperature,area: mean time: point,TIER1,
+tsl,mon,K,Temperature of Soil,soil_temperature,area: mean where land time: mean,TIER1,Temperature of each soil layer (3D variable). Reported as missing for grid cells with no land.
+tsl,day,K,Temperature of Soil,soil_temperature,area: mean where land time: mean,TIER1,Temperature of each soil layer (3D variable). Reported as missing for grid cells with no land.
+tsl,6hr,K,Temperature of Soil,soil_temperature,area: mean where land time: point,TIER1,Temperature of each soil layer (3D variable). Reported as missing for grid cells with no land.
```

Comment:
High frequency output from slow-varying fields are good candidates to drop from the data request, especially for 3D fields (tsl, mrsfl, mrsol)